tests/e2e/tests/pilot/testdata/config.yaml.tmpl

```diff
@@ -27,8 +27,8 @@ data:
     mixerReportServer:        istio-mixer.{{.IstioNamespace}}:15004
 {{end}}
     authPolicy:               {{ .Auth }}
-    connectTimeout:           1s
-    rdsRefreshDelay:          1s
+    connectTimeout:           5s
+    rdsRefreshDelay:          4s
```

Comment:
Why? This is going to fail every single config test at the first retry.

Reply:
The tests seem to be passing with 4 sec - and yes, they may fail on the first retry, as they should. 

We have to increase the refresh - since it cant' scale, and tests should reflect this.